src/OrleansLoggerProvider/OrleansLoggingDecorator.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+
+namespace Orleans.Runtime
+{
+
+    /// <summary>
+    /// Config for message bulking feature
+    /// </summary>
+    public class MessageBulkingConfig
```

Comment:
how about `MessageBulkingOptions` (or `EventBulkingOptions` if you also rename the decorator that way)?

Reply:
I think `EventBulkingConfig` is good. Why do you prefer **options** over **config**, as in I think this is a config class?

Reply:
with the new builders, we are moving to using strongly typed Options (that could be bound using Microsoft.Extensions.Options). We'll get rid of our legacy Config and Configuration classes, and morph them to Options (meaning we'll remove any parsing logic from them too, something similar to this message bulking options)